resources/assets/js/app/views/users/userCollection.js

```diff
@@ -0,0 +1,8 @@
+var Marionette = require('backbone.marionette');
+
+module.exports = Marionette.CollectionView.extend({
+    template: '#users',
```

Comment:
Why do you think so? This is correct layaer id from [resources/assets/templates/user/userLayout.hbs](https://github.com/BinaryStudioAcademy/bsa-2016-forum/pull/78/commits/0fe712f7f1bb1eb93981b6c412169ac781577231#diff-d6ca02238c39872db132e61890ea637dR2)